app/data/mounts.json

```diff
@@ -322,6 +336,19 @@
             "spellid": 336036,
             "icon": "inv_maldraxxusflyermount",
             "itemId": 181819
+          },
+          {
+            "ID": 1437,
+            "name": "Battle-Bound Warhound",
```

Comment:
This doesn't drop from a dungeon, this drops from the non-instanced bosses of Theater of Pain.

Reply:
Is now grouped with the maw mount under "Zone". Not a zone wide drop, but both drop in a zone. I couldn't come up with something better, but I didn't want these mounts to have their own category.